### PR TITLE
Add support for SKIZ format

### PR DESCRIPTION
**Pull request by kirkeaton**
Add support for the SKIZ format which is the format is used by the [Ski Tracks](https://www.skitracks.com/) mobile app.

Sample file can be downloaded here: https://github.com/kirkeaton/skiz-parser/raw/main/test/example.skiz

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -23,5 +23,8 @@ module.exports = {
                 }
             }
         ]
+    },
+    node: {
+        fs: 'empty'
```
Comment (kirkeaton) [inline webpack.config.js:28]: This was added to resolve errors with webpack after adding `skiz-parser` as a dependency.

File: src/track.js [hunk 2]
```diff
@@ -144,6 +145,23 @@ function extractIGCTracks(igc) {
...
+        points.push({
+            lat: node.latitude,
+            lng: node.longitude,
+        });
+
+        node.timestamp && (timestamp = node.timestamp);
+    }
+
+    const name = 'skiz';
+    return points.length > 0 ? [{timestamp, points, name}] : []
```
Comment (erik) [inline src/track.js:162]: tiny nit, could you add a semicolon here? below on 246 as well

**Review by erik — APPROVED** [review]
Thanks @kirkeaton! Never heard of SKIZ before.

Seems like a pretty minimal addition, I'd be happy to include this.

File: src/track.js (hunk 2 above)
Comment (kirkeaton) [reply src/track.js]: done!

**Pull request merged**